src/idv_service/session/create/face_capture/upload.face.capture.image.payload.builder.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const Validation = require('../../../../yoti_common/validation');
+const UploadFaceCaptureImagePayload = require('./upload.face.capture.image.payload');
+
+/**
+ *
+ * @class UploadFaceCaptureImagePayloadBuilder
+ */
+class UploadFaceCaptureImagePayloadBuilder {
+  /**
+   * Sets the content type for uploading a JPEG image
+   *
+   * @return this
+   */
+  forJpegImage() {
```

Comment:
Do we really need these `for` methods?

Just wondering if that could become a problem for a consumer.

i.e they use an image like `cat.png` and accidentally use the `.forJpegImage()`. What would happen?

One alternative to these methods is to instead to use https://www.npmjs.com/package/file-type, to detect the file-type from the image buffer itself. That way you are kinda removing the need for these methods. Then it would just be a case of checking if the mimeType is a valid one (image/jpg or image/png).

Reply:
I do agree with you and I wanted to implement an internal file type checker instead of these methods, but I think we need to follow the pattern of the existing same PRs for other languages.